ThirdPartyAdapters/applovin/applovin/src/main/java/com/applovin/mediation/AppLovinNativeAdListener.java

```diff
@@ -0,0 +1,107 @@
+package com.applovin.mediation;
+
+import android.content.Context;
+import android.util.Log;
+
+import com.applovin.nativeAds.AppLovinNativeAd;
+import com.applovin.nativeAds.AppLovinNativeAdLoadListener;
+import com.applovin.nativeAds.AppLovinNativeAdPrecacheListener;
+import com.applovin.sdk.AppLovinSdk;
+import com.applovin.sdk.AppLovinSdkUtils;
+import com.google.android.gms.ads.AdRequest;
+import com.google.android.gms.ads.mediation.MediationNativeListener;
+
+import java.lang.ref.WeakReference;
+import java.util.List;
+
+public class AppLovinNativeAdListener
```

Comment:
You don't need to make this public for the world right? Since it' sonly access within this package, you can make the scope smaller by removing (public) to default into package-level access.